### ISSUE TITLE
gpu mining turtlecoin .. not working ? (0 shares)

### ISSUE THREAD

**Issue by lostence** [turtlecoin/violetminer#42]
Hi
I tried to GPU mine TRTL with my GTX 770 and violetminer.
I --disableCPU and I get absolutely 0% shares through.. also reported hashrate is too high to be true :

>[turtlecoin.herominers.com:10381] Attempting to connect to pool...
>[turtlecoin.herominers.com:10381] Logged in.
>Resuming mining.
>[turtlecoin.herominers.com:10381] New job, diff 250012
>[GPU 0] Intensity: 100, Desktop Lag: 100
>[GPU 0] Sleeping for 0 seconds between kernel launches (0 microseconds)
>[GPU 0] Allocating 1.625GB of GPU memory.
>[GPU 0] Performing 1664 iterations per kernel launch, with 16 jobs per block.
>[turtlecoin.herominers.com:10381] GeForce GTX 770-0   | 65507048.62 H/s
>[turtlecoin.herominers.com:10381] Accepted Shares     | 0.00%
>[turtlecoin.herominers.com:10381] GeForce GTX 770-0   | 64873578.20 H/s
>[turtlecoin.herominers.com:10381] Accepted Shares     | 0.00%`

Is it doable or not ?
I tried reverting to older nvidia drivers but it doesnt change anything ..
Thanks

**Comment by crappyrules** [comment]
Try a different pool

**Comment by lostence** [comment]
same problem on pool.hashvault.pro .. i don't think it depends on pool?

**Comment by 0des** [comment]
That hashrate doesnt look right.  Does this only happen when you disable CPU?

**Comment by zpalmtree** [comment]
Violetminer ships with cuda 11.1, which unfortunately does not support the gtx770. I believe the latest cuda that will work for that GPU is Cuda 9.1. 
https://en.wikipedia.org/wiki/GeForce_700_series#Discontinued_support

You could try compiling yourself with cuda 9.1 to get a working build, but CUDA is a real pain to get working, in addition I believe you would need to downgrade the visual studio version - It's doable, but not something I'm prepared to support myself.

**Comment by lostence** [comment]
Ok thanks for the clarification, that makes sense.
As i could install the latest nvidia driver which is supposed to run cuda 11.1, i thought it should work.

But that's more complicated than that apparently 👍 
Thank you

**Comment by pavargr** [comment]
Having 7 old pcs with gtx 750 ti 2GB, after 2 days o frustration I ended up building a cuda 10.2 version based on original 0.2.2.  
I have tested it with win 7 64bit with amd fx4300 CPU in 2 different pools (herominers and fastpool) and it works fine.  hashrate is between 0.5%-1.5% more than using xmrig for the same job.
The original code remains as is, beyond adding a few variables to allow it to work with cuda 10.2 as it was never intended to do so,  Also, devfee has been re-enabled to 1%.  
Here is the [link](https://mega.nz/file/jN90Fa7Z#_ddqK5ED5OfO_iZzUAdSm5h4Cm1N-JNc9dqV99By-n0)
